### DISCUSSION TITLE
Queue Printer upload settings

### DISCUSSION THREAD

**Discussion by William-i-am** [supermerill/SuperSlicer#1697] [Q&A] [answered]
Are there settings for a queue printer I'm running Network printers and Octoprint servers would be nice to have?  _**I thought I have seen it**_ but could not find it again must have been dreaming.

**Comment by supermerill** [comment, marked as answer]
you have to clic on the cog in the printer config tab, it's near the save icon on the top.
It will prompt you how to connect to your physical printer.
You can select all the profiles to link with this printer.
then, after slicing, you will have a upload button to send & start the gcode directly.

**Reply by William-i-am** [reply]
Thank you
